CONTRIBUTING.adoc

```diff
@@ -102,3 +102,33 @@ $ git merge fix-123
 # Push to origin
 $ git push origin master
 -------------------------------------------------------------------------
+
+== Releasing
+
+* Review https://github.com/projectatomic/vagrant-service-manager/milestones[milestone] for the current release
+** Make sure all issues are resolved
+** Move unresolved issues into an upcoming release
+** Close milestone
+* Update the link:CHANGELOG.adoc[CHANGELOG]
```

Comment:
Let's maybe discuss this a bit later. Seems we have not reached a consensus here.